docs/changelog/87361.yaml

```
@@ -3,4 +3,17 @@ summary: "Implement per-transform num_failure_retries setting"
 area: Transform
 type: enhancement
 issues: []
+highlight:
+  title: Infinite and adaptive retries for Transforms
```

I don't think we generally capitalize this feature:
```suggestion
  title: Infinite and adaptive retries for transforms
```